src/index.css

```diff
@@ -10,6 +17,12 @@ body {
   margin: 0;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(140, 41, 241);
+  background: linear-gradient(
+    90deg,
+    rgba(140, 41, 241, 1) 0%,
+    rgba(131, 85, 255, 1) 100%
```

Comment:
This one is `#8355ff`, maybe `--accent`?

Reply:
Changed